OpenRA.Game/FileSystem/BagFile.cs

```diff
@@ -39,14 +39,14 @@ public BagFile(string filename, int priority)
 
 			// Build the index and dispose the stream, it is no longer needed after this
 			List<IdxEntry> entries;
-			using (var indexStream = GlobalFileSystem.Open(indexFilename))
+			using (var indexStream = Game.ModData.ModFiles.Open(indexFilename))
```

Comment:
The next step will be to pass the `ModFiles` as an argument in the ctor.  Fixing that isn't required for my :+1:, but IMO you may as well do it now.